provisioning_templates/provision/kickstart_default.erb

```diff
@@ -43,7 +43,15 @@ This template accepts the following parameters:
   chef_enabled = @host.respond_to?(:chef_proxy) && @host.chef_proxy
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
   use_ntp = host_param_true?('use-ntp') || (is_fedora && os_major < 16) || (rhel_compatible && os_major <= 7)
+  iface = @host.provision_interface
 %>
+
+<%- if iface.tag.present? &&  iface.attached_to.present?
+  boot_options = "--device=#{iface.attached_to} --vlanid=#{iface.tag}"
+else
+  boot_options = os_major >= 6 ? " --device=#{@host.mac}" : ''
+end -%>
```

Comment:
I suggest to do similar approach as above - make `boot_options` an array and then do `boot_options.join(' ')` on the `network` line. Probably the variable should be named something like `network_options` which fits better.